webserver/grpc/pom.xml

```diff
@@ -43,6 +43,14 @@
             <groupId>io.helidon.webserver</groupId>
             <artifactId>helidon-webserver-http2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.helidon.grpc</groupId>
+            <artifactId>helidon-grpc-core</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
Please reconsider the tracing approach. It should not be "hardcoded" dependency.